modules/delete-all-autogenerated-policies.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_module-type: PROCEDURE
+[id="delete-all-autogenerated-policies_{context}"]
+= Deleting all autogenerated policies
+
+[role="_abstract"]
+You can delete all generated policies from your cluster that you have created by using {product-title}.
+
+.Procedure
+
+* Run the following command:
+** On {ocp}:
++
+[source,terminal]
+----
+$ oc get ns -o jsonpath='{.items[*].metadata.name}' | xargs -n 1 oc delete networkpolicies -l 'network-policy-generator.stackrox.io/generated=true' -n
```

Comment:
Just checking that the `-n` at the end of these commands isn't expecting something to be passed in (like the namespace).

Reply:
Thanks @bergerhoffer Yes, I confirmed, the command is correct as it is. The namespace is passed by `xargs`.